components/Collections/BUILD

```diff
@@ -39,6 +41,23 @@ mdc_public_objc_library(
     ],
 )
 
+mdc_examples_objc_library(
+    name = "ObjcExamples",
+    deps = [
+        ":Collections",
+        "//components/Palettes",
+        "//components/Typography",
+    ],
+)
+
+mdc_examples_swift_library(
+    name = "SwiftExamples",
+    deps = [
+        ":Collections",
+    ],
+)
+
+
```

Comment:
Nit: Please remove the extra line. You can also run `buildifier` before committing to save you time.